cocotb/share/lib/fli/FliImpl.h

```diff
@@ -418,7 +418,6 @@ class FliTimerCache {
 class FliIterator : public GpiIterator {
 public:
     enum OneToMany {
-        OTM_END = 0,
         OTM_CONSTANTS,  // include Generics
```

Comment:
Not sure if `OTM_CONSTANTS` now needs to be `1`, or if this is a true enum whose values are meaningless.

Reply:
It doesn't cause any issue in the regression and looking at the code I don't see why it would be an issue.